jaxrs/jaxrs-deltaspike-data/src/test/java/it/jaxrs/PersonRestIT.java

```diff
@@ -63,7 +52,7 @@ public static Archive createDeployment() throws Exception {
     @RunAsClient
     public void shouldGetAll(){
         Response response = target.request().get();
-        assertThat(response.getStatus()==200);
+        assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
```

Comment:
Please note - without these changes, the tests were actually failing, but ignored since its using fest assert.